File: src/main/java/com/example/app/service/EmailService.java
package com.example.app.service;

import jakarta.mail.MessagingException;

public interface EmailService {

    void activateUserByEmail(String emailTo,String login, String activationCode) throws MessagingException;
    void recoveryPasswordByEmail(String emailTo, String login, String activationCode) throws  MessagingException;

}

File: src/main/kotlin/ru/kaplaan/service/EmailService.kt
package ru.kaplaan.service

import jakarta.mail.MessagingException
import org.springframework.stereotype.Service

@Service
interface EmailService {
    fun activateUserByEmail(emailTo: String, login: String, activationCode: String)

    fun recoveryPasswordByEmail(emailTo: String, login: String, activationCode: String)
}
